components/contact-page/contact-form/ContactForm.js

```diff
@@ -13,13 +13,13 @@ export default function ContactForm({ togglePopup }) {
     <>
       <ContactFormContainer>
         <InputFieldWrapper>
-          <TextInputField id="firstName" label="First Name" />
+          <TextInputField id="firstName" label="First Name" required={true} />
```

Comment:
It's good either way, but for boolean props you can omit the `={true}`. If it's present like ` <TextInputField id="firstName" label="First Name" required />` It'll be true, and otherwise false.

Reply:
Yes planning on making error checking / blocking submission a separate PR!